api/travel.php

```diff
@@ -3,7 +3,7 @@
 # Begin standard auth
 require_once __DIR__ . "/../classes.php";
 
-$system = API::init();
+$system = API::init(row_lock: false);
```

Comment:
```suggestion
$system = API::init(row_lock: true);
```
I think we need to enable locking for this one, high likelihood of causing conflicts